File: fotocopiadora.rb
system("clear") || system("cls")
################################
print "ingresa la cantidad de hojas por favor"
hojas = gets.chomp.to_i
#################################
precio_por_hoja = 20
#################################
total = precio_por_hoja * hojas
##################################
total *= 0.90 if hojas > 30
###################################
puts "presiona enter para continuar"
####################################
gets
####################################
system("clear") || system("cls")
######################################
puts <<~Boleta
*********Boleta********
cantidad de hojas #{hojas}
valor por hoja $#{precio_por_hoja}
total $#{total.round(2)}
Boleta
#######################################

File: fotocopiadora.exs
defmodule fotocopiadora do 
    def main do 
      IO.puts("\e[H\e[2J")
#######################################
      hojas =
        IO.gets("ingresa la cantidad de hojas por favor")
        |> String.trim()
        |> String.to_integer()
#######################################
        total = hojas * 20
########################################
        total = if hojas > 30, do: total * 0.90, else: total
#########################################
        IO.puts("presiona enter para continuar")
        IO.gets("")
#########################################
        IO.puts("\e[H\e[2J")
#########################################
        IO.puts("*****Boleta****
        cantidad de hojas #{hojas}
        valor por hoja $20
        TOTAL $#{total}")
      end
end
##########################################
fotocopiadora.main()
##########################################                                                   
